hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/ByteArrayObjectDataOutput.java

```diff
@@ -386,6 +386,12 @@ public void writeData(Data data) throws IOException {
         }
     }
 
+    @Override
+    public void writeObjectAsData(Object object) throws IOException {
```

Comment:
Is this only used in tests?

Reply:
Hm.. that was meant as the equivalent of `input.readDataAsObject` but after introduction of `DeferredValue` this is no longer needed so will be removed.